src/include/daos_prop.h

```diff
@@ -190,8 +190,8 @@ daos_svc_rf_is_valid(uint64_t svc_rf)
  * Level of perf_domain, should be same value as PO_COMP_TP_xxx (enum pool_comp_type).
  */
 enum {
-	DAOS_PROP_PERF_DOMAIN_ROOT = 255,
-	DAOS_PROP_PERF_DOMAIN_GROUP = 3,
+	DAOS_PROP_PERF_DOMAIN_ROOT  = 255,
+	DAOS_PROP_PERF_DOMAIN_GROUP = 200,
```

Comment:
Should these use the enum names from pool_map.h instead of raw values?

Reply:
Yeah, I felt that way too. I think they are defined this way to avoid circular dependencies. daos/pool_map.h includes daos/common.h, which includes daos_prop.h. Not sure if it's an over-broad include for convenience or if there is really something in daos_prop.h that is needed by daos/pool_map.h.

I usually take stuff like this as a sign that a header file needs to be broken up (daos/common.h maybe shouldn't exist). Feels out of scope of this PR though.